LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 HiQ Finland
```

Comment:
Change the owner to HiQ Finland / FRENDS.

Reply:
Fixed